pinecone/local_test.go

```diff
@@ -0,0 +1,154 @@
+//go:build localServer
```

Comment:
Isolating these tests from the other tests. I think this works for not running these tests as a part of the unit / integration test suite, but there may be a better way to do this in Go.